Source/WebInspectorUI/UserInterface/Views/LocalResourceOverridePopover.js

```diff
@@ -532,14 +543,19 @@ WI.LocalResourceOverridePopover = class LocalResourceOverridePopover extends WI.
             mimeTypeRow.element.hidden = isRequest || isBlock;
             statusCodeRow.element.hidden = isRequest || isBlock;
             headersRow.hidden = isBlock;
-            if (optionsRowElement)
-                optionsRowElement.hidden = isRequest || isBlock;
+
+            isPassthroughLabel.hidden = isBlock;
+            if (skipNetworkLabel)
+                skipNetworkLabel.hidden = isRequest || isBlock;
+            optionsRowElement.hidden = isPassthroughLabel.hidden && (skipNetworkLabel?.hidden ?? true);
 
             if (isRequest) {
                 this._requestURLCodeMirror.refresh();
 
                 if (contentTypeDataGridNode.parent)
                     this._headersDataGrid.removeChild(contentTypeDataGridNode);
+
+                isPassthroughLabelText.textContent = WI.UIString("Include original request data");
             }
 
             if (isResponse) {
```

Comment:
i'd personally like to keep this as-is in case we decide to add a way for a single local override to affect both request and response in the future, but i recognize that there will likely have to be other things that change then too (e.g. need more than one "Headers" table) so i'll change